lib/component.js

```diff
@@ -32,11 +32,19 @@
   if (typeof define === 'function' && define.amd) {
     define(['react', 'backbone', 'underscore'], factory);
   } else if (typeof module !== 'undefined' && module.exports) {
-    module.exports = factory(require('react'), require('backbone'), require('underscore'));
+    var react = null;
+    var isReactNative = false;
+    try {
+      react = require('react');
+    } catch (error) {
+      react = require('React'); // React Native
```

Comment:
I've created a while ago a `React Native` example project and the way the library is required is with `require('react-native')`. Are you sure about `React`?